pills/04-basics-of-language.xml

```diff
@@ -60,25 +60,32 @@
 
     <para>
       Launch <literal>nix repl</literal>. First of all, Nix supports basic arithmetic operations:
-      <literal>+</literal>, <literal>-</literal>, and <literal>*</literal>. Integer division can be
-      done with <literal>builtins.div</literal>.
+      <literal>+</literal>, <literal>-</literal>, <literal>*</literal> and <literal>/</literal>.
       (To exit <literal>nix repl</literal>, use the command <literal>:q</literal>.
       Help is available through the <literal>:?</literal> command.)
     </para>
 
     <screen><xi:include href="./04/basics.txt" parse="text" /></screen>
 
     <para>
-      You might wonder why Nix doesn't have basic operations such as integer division. The answer is
-      because it's not needed for creating packages. Nix is not a general purpose
-      language, it's a domain-specific language for writing packages.
+      Attempting to perform division in Nix can lead to some surprises.
     </para>
 
+    <screen><xi:include href="./04/relative-path.txt" parse="text" /></screen>
+
     <para>
-      Just think - <literal>builtins.div</literal> is not used in the whole of the
-      nixpkgs repository: it's not actually useful if you are writing package expressions.
+      What happened?
+      Recall that Nix is not a general purpose
+      language, it's a domain-specific language for writing packages.
+      Integer division isn't actually that useful when writing
+      package expressions.
```

Comment:
Can you re-indent the lines around 80 characters?

Reply:
Done